ontology/yaml/resources/SAFETY/entity_types/FACP.yaml

```diff
@@ -33,4 +33,11 @@ FACP_FA2X:
   - FA2X
   opt_uses:
   - panel_alarm
+  
+FACP_GASDET:
```

Comment:
shouldnt there be a fire alarm on the fire alarm control panel?

Reply:
Yes... Used an existing cannonical

FACP_GASDET:
  description: "Fire alarm panel with gas leak detection alarm"
  is_canonical: true
  implements:
  - FACP
  - GASDET

Reply:
Deleted this canonical since the existing canonical of LDS_GASDET is used